snappy/overlord.go

```diff
@@ -274,21 +345,11 @@ func DeactivateSnap(s *Snap, inter interacter) error {
 		return err
```

Comment:
Should the logic above be replaced by RemoveGeneratedWrappers?

Should it continue attempting to remove the follow up steps even if the first step fails? (solved if using the other func)